projects/packages/connection/src/class-manager.php

```diff
@@ -863,6 +867,22 @@ public function connect_user( $user_id = null, $redirect_url = null ) {
 		exit();
 	}
 
+	/**
+	 * Force user disconnect.
+	 *
+	 * @param int $user_id Local (external) user ID.
+	 *
+	 * @return bool
+	 */
+	public function disconnect_user_force( $user_id ) {
```

Comment:
Nitpick but `force_disconnect_user` sounds a bit more intuitive :)

Reply:
Yeah in this case the action would be `force_disconnect` (verb), but anyway it's definitely not a blocker.